src/html-to-ast.js

```diff
@@ -78,6 +78,10 @@ export function parse(html) {
                 node.range = range;
                 node.raw = html.slice(range[0], range[1]);
             }
+            // map `url` to Link node
+            if (node.type === "Link" && node.properties.href !== "undefined") {
```

Comment:
> `node.properties.href !== "undefined"`

`typeof node.properties.href !== "undefined"` ?

Can you add a testcase for `<a>text</a>` pattern?

Reply:
Thanks for the review. I'll fix it.